### PR TITLE
relevance docs

### PR DESCRIPTION
**Pull request by noamzbr**

### CODE REVIEW

File: docs/source/nlp/usage_guides/nlp_properties_extended.rst [hunk 1]
```diff
@@ -163,3 +164,52 @@ Michael Jordan (1963) is an American former professional basketball player and b
...
+The reason for taking the mean of the Max Word Similarity and the minimum of the Max Entity Similarity is that for
+entities we expect to find a high match for each one, or else the output is not relevant enough. However, for regular
+words, we expect to find a match for only some of them, and the rest can be ignored.
+
+`Back to Property List <#properties>`__
+
+Examples
+~~~~~~~~
+======================================================================================================================  =====================================================  ===================
+User Input                                                                                                               LLM Output                                             Grounded in Context
```
Comment (nirhutnik) [inline docs/source/nlp/usage_guides/nlp_properties_extended.rst:211]:
```suggestion
User Input                                                                                                               LLM Output                                             Relevance
```

File: docs/source/nlp/usage_guides/nlp_properties_extended.rst [hunk 2]
```diff
@@ -163,3 +164,52 @@ Michael Jordan (1963) is an American former professional basketball player and b
...
+======================================================================================================================  =====================================================  ===================
+What is the color of the sky?                                                                                           The sky is blue                                        0.81
+Where was the first World War II attack in the United States?                                                           Pearl Harbor                                           0.84
```
Comment (nirhutnik) [inline docs/source/nlp/usage_guides/nlp_properties_extended.rst:214]: If these are the actual values, maybe update them in the description?

File: docs/source/nlp/usage_guides/nlp_properties_extended.rst [hunk 3]
```diff
@@ -163,3 +164,52 @@ Michael Jordan (1963) is an American former professional basketball player and b
 Michael Jordan (1963) is an American former professional basketball player and businessman. When was Michael born?      Michael Jeffrey Jordan was born in 1963                0.5
 Michael Jordan (1963) is an American former professional basketball player and businessman. What did he achieve?        He won many NBA championships with the Chicago Bulls   0.0
 ======================================================================================================================  =====================================================  ===================
+
+
+Relevance
+---------
+
+The Relevance score is a measure of how relevant the LLM output is to the input, ranging from 0 (not relevant) to 1
+(fully relevant).
```
Comment (nirhutnik) [inline docs/source/nlp/usage_guides/nlp_properties_extended.rst:173]: Can I not read the entire thing and just tell me what you changed? 😅

Comment (noamzbr) [reply docs/source/nlp/usage_guides/nlp_properties_extended.rst:173]: lol not a lot

**Comment by noamzbr** [comment]
Already done

**Pull request closed without merging**